<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseView</class>
 <widget class="QWidget" name="LicenseView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>635</width>
    <height>453</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>635</width>
    <height>453</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;签发升级包&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>20</width>
     <height>351</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="timeSpanCheckBox">
      <property name="text">
       <string>时间跨度：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>追加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>减少</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_8">
        <property name="text">
         <string>禁用</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLineEdit" name="timeSpanLineEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="timeSpanComboBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>天</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>秒</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>241</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QCheckBox" name="overTimeCheckBox">
      <property name="minimumSize">
       <size>
        <width>239</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>239</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>结束时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QRadioButton" name="radioButton_13">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_14">
        <property name="text">
         <string>追加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_15">
        <property name="text">
         <string>减少</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_16">
        <property name="text">
         <string>禁用</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QDateEdit" name="overTimeDateEdit"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="overTimeTimeEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>241</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QCheckBox" name="useCountCheckBox">
      <property name="text">
       <string>使用计数：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QRadioButton" name="radioButton_17">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_18">
        <property name="text">
         <string>追加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_19">
        <property name="text">
         <string>减少</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_20">
        <property name="text">
         <string>禁用</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLineEdit" name="useCountLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>82</width>
     <height>189</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="fileButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="licenseButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>许可</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clockButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>时钟</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="seedButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>种子</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>410</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="previewButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>预览</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>签发升级包</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>241</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCheckBox" name="startTimeCheckBox">
      <property name="minimumSize">
       <size>
        <width>239</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>239</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>开始时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>追加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>减少</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>禁用</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QDateEdit" name="startTimeDateEdit"/>
      </item>
      <item>
       <widget class="QTimeEdit" name="startTimeTimeEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>许可类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="licenseTypeComboBox">
      <item>
       <property name="text">
        <string>许可增加</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>许可更新</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>许可删除</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>锁定所有许可</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>解锁所有许可</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>删除所有许可</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QCheckBox" name="readonlyCheckBox">
      <property name="text">
       <string>只读区</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="readonlyButton">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QCheckBox" name="readWriteCheckBox">
      <property name="text">
       <string>读写区</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="readWriteButton">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QCheckBox" name="publicCheckBox">
      <property name="text">
       <string>公开区</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="publicButton">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>379</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>并发方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QRadioButton" name="radioButton_9">
          <property name="text">
           <string>单机</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_12">
          <property name="text">
           <string>网络</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QComboBox" name="concurrentManerComboBox">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>进程</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>会话</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="concurrentManerLineEdit">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="licenseCheckBox">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>许可ID：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="licenseIDLineEdit">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="perpetualLicenseCheckBox">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>永久许可</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
